frontends/concrete-python/concrete/fhe/tfhers/context.py

```diff
@@ -0,0 +1,192 @@
+                return fhe.Value(TfhersExporter.import_fheuint8(buffer, keyid, variance))
+
+        msg = (
+            f"importing {'signed' if signed else 'unsigned'} integers of {bit_width}bits is not"
+            " yet supported"
+        )
+        raise NotImplementedError(msg)
+
+    def export_value(self, value: "fhe.Value", output_idx: int) -> bytes:
+        """Export a value as a serialized TFHErs integer.
+
+        Args:
+            value (fhe.Value): value to export
+            output_idx (int): the index corresponding to this output
+
+        Returns:
+            bytes: serialized fheuint8
+        """
+        output_type = self._output_type(output_idx)
+        if output_type is None:  # pragma: no cover
+            msg = "output at 'output_idx' is not a TFHErs value"
+            raise ValueError(msg)
+
+        # construct a TFHErs integer description based on type
+        bit_width = output_type.bit_width
+        signed = output_type.is_signed
+        params = output_type.params
+        message_modulus = 2**output_type.msg_width
+        carry_modulus = 2**output_type.carry_width
+        lwe_size = params.polynomial_size + 1
+        n_cts = bit_width // output_type.msg_width
+        ks_first = params.encryption_key_choice is EncryptionKeyChoice.BIG
+        # TODO: how to set degree and noise_level
+        degree: int = 0
+        noise_level: int = 0
+        fheint_desc = TfhersFheIntDescription.new(
+            bit_width,
+            signed,
+            message_modulus,
+            carry_modulus,
+            degree,
+            lwe_size,
+            n_cts,
+            noise_level,
+            ks_first,
+        )
+
+        if bit_width == 8:
+            if not signed:
+                return TfhersExporter.export_fheuint8(value.inner, fheint_desc)
+
+        msg = (
+            f"exporting value to {'signed' if signed else 'unsigned'} integers of {bit_width}bits"
+            " is not yet supported"
+        )
+        raise NotImplementedError(msg)
+
+    def serialize_input_secret_key(self, input_idx: int) -> bytes:
+        """Serialize secret key used for a specific input.
+
+        Args:
+            input_idx (int): input index corresponding to the key to serialize
+
+        Returns:
+            bytes: serialized key
+        """
+        keyid = self._input_keyid(input_idx)
+        input_type = self._input_type(input_idx)
+        if input_type is None:  # pragma: no cover
+            msg = "input at 'input_idx' is not a TFHErs value"
+            raise ValueError(msg)
+        glwe_dim = input_type.params.glwe_dimension
+        poly_size = input_type.params.polynomial_size
+        # pylint: disable=protected-access
+        return self.circuit.client.keys._keyset.serialize_lwe_secret_key_as_glwe(  # type: ignore
+            keyid, glwe_dim, poly_size
+        )
+        # pylint: enable=protected-access
+
+
+def new_context(
```

Comment:
Why this wrapper around the constructor?

Reply:
a matter of taste :stuck_out_tongue:

Reply:
Is that python idomatic?